kernelci/kbuild.py

```diff
@@ -528,18 +594,33 @@ def upload_artifacts(self):
                 (artifact_path, artifact), root_path
             )
             artifact_key = self.map_artifact_name(artifact)
-
             node_af[artifact_key] = stored_url
+            # map also bzImage, zImage, Image to kernel
+            # for template simplicity
+            if artifact in KERNEL_IMAGE_NAMES[self._arch]:
+                node_af['kernel'] = stored_url
```

Comment:
```suggestion
            # map also bzImage, zImage, Image to kernel
            # for template simplicity
            if artifact in KERNEL_IMAGE_NAMES[self._arch]:
                node_af['kernel'] = stored_url
            else:
                node_af[artifact_key] = stored_url
```

This would avoid having both `Image` and `kernel` in the artifacts list.

Reply:
I left both intentionally, as i'm wondering if in some cases we need to check image type present. But if you think it is not necessary, i can drop that and do as you suggesting.

Reply:
I think the image type should be deduced from some other parameter (e.g. if the platform config has a `dtb` key), and with #2319 merged, I think we should move forward with change. Unless `bzImage`/`Image` is used elsewhere?